test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs

```diff
@@ -227,8 +242,7 @@ private static Task CheckAppCompressionAsync(HttpClient client, ILogger logger)
         private static string GetContentLength(HttpResponseMessage response)
         {
             // Don't use response.Content.Headers.ContentLength, it will dynamically calculate the value if it can.
-            IEnumerable<string> values;
-            return response.Content.Headers.TryGetValues(HeaderNames.ContentLength, out values) ? values.FirstOrDefault() : null;
+            return response.Content.Headers.TryGetValues(HeaderNames.ContentLength, out var values) ? values.FirstOrDefault() : null;
```

Comment:
If we are using c#7 for this repository, we can ignore values like
``` out _```

Reply:
We're using the variable.